rfc-template.md

```diff
@@ -0,0 +1,67 @@
+- Contribution Name: (fill me in with a unique ident, `my_awesome_feature`)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- Target Date: (is there a target date of completion?)
+- RFC PR: [linkerd/rfc#0000](https://github.com/linkerd/rfc/pull/0000)
+- Linkerd Issue: [linkerd/linkerd2#0000](https://github.com/linkerd/linkerd2/issues/0000)
+- Reviewers: (Who should review the code deliverable? ex. @olix0r)
+
+# Summary
+
+[summary]: #summary
+
+One paragraph explanation of the proposed contribution.
+
+# Motivation
```

Comment:
- Use Cases
- Goals
- Non-Goals

Reply:
👍 for "non-goals", it's good to state this clearly.